Examples/MAX78000/CNN/kws20_demo/main.c

```diff
@@ -263,6 +275,45 @@ void WUT_IRQHandler()
 }
 #endif
 
+int console_UART_init(uint32_t baud)
```

Comment:
Doesn't the board file already initialize the console UART?

Reply:
It does with 115k.  This is placed to enable optionally higher baudrate for serial bin transfer.

Reply:
Understood.  Thanks for the clarification.